### PR TITLE
FX-3130 feat: Fair organizer page top level details

### PR DESCRIPTION
**Pull request by anastasiapyzhik**
[FX-3130]

[FX-3130]: https://artsyproduct.atlassian.net/browse/FX-3130

This PR adds overview of fair organizer and upcoming fair on page

<details>
<summary>Desktop</summary>

Screenshot 2021-08-06 at 12 51 36
</details>

<details>
<summary>Mobile</summary>

Screenshot 2021-08-06 at 12 52 34
</details>

### CODE REVIEW

**Comment by damassi** [comment]
@anastasiapyzhik - since this is a draft PR it might be too early to comment, so please forgive if so! Reminder to fill out the PR description body (vs just dropping in a link to JIRA). Some devs might not have full access to the JIRA project, and we'd also like to avoid storing PR context in a 3rd party.

**Comment by anastasiapyzhik** [comment]
> @anastasiapyzhik - since this is a draft PR it might be too early to comment, so please forgive if so! Reminder to fill out the PR description body (vs just dropping in a link to JIRA). Some devs might not have full access to the JIRA project, and we'd also like to avoid storing PR context in a 3rd party.

Thanks for the reminder! Was gonna add a description after completing the task but let it be right now 🙂

**Review by dimatretyak — APPROVED** [review]
LGTM 👍

File: src/v2/Components/Timer.tsx [hunk 1]
```diff
@@ -39,8 +48,13 @@ export const Timer: React.FC<{
         const hasEnded = Math.floor(duration.seconds) <= 0
 
         return (
-          <Flex flexDirection="column" alignItems="center">
+          <Flex flexDirection="column" {...rest}>
```
Comment (dblandin) [inline src/v2/Components/Timer.tsx:51]: **props**: Nice change to thread other Flex props through 👍🏼

**Review by dblandin — APPROVED** [review]
This looks great! Nice work! 💯

**Pull request merged**